tests/e2e/koperator_suite_test.go

```diff
@@ -59,11 +59,10 @@ var _ = When("Testing e2e test altogether", Ordered, func() {
 	testInstall()
 	testInstallZookeeperCluster()
 	testInstallKafkaCluster("../../config/samples/simplekafkacluster.yaml")
-	testProduceConsumeInternal()
 	testUninstallKafkaCluster()
 	testInstallKafkaCluster("../../config/samples/simplekafkacluster_ssl.yaml")
-	testProduceConsumeInternalSSL(defaultTLSSecretName)
```

Comment:
Not sure if I have all the context here - why are we removing the consumer tests via internal listeners?

Reply:
oh boy... I did not mean for that. My best guess at the moment is that I made a mistake when I solved a conflict during a rebase.

I will fix this.  
Thank you for catching it!